pkg/sql/opt/xform/testdata/rules/project

```diff
@@ -17,7 +17,7 @@ CREATE TABLE a (
 # Eliminate the IndexJoin when the Project passthrough columns are a subset of
 # the IndexJoin's input columns.
 opt expect=EliminateIndexJoinInsideProject
-SELECT k, i FROM a WHERE s = 'foo'
+SELECT k, i FROM a WHERE s LIKE 'foo%'
```

Comment:
I had to change the predicate so that the index join wasn't replace with a projection of the constant `foo`.